subtitles/zh-CN/31_navigating-the-model-hub.srt

```diff
@@ -20,42 +20,42 @@ This is the huggingface.co landing page.
 
 5
 00:00:13,260 --> 00:00:16,020
-要访问模型中心，请单击模型选项卡
+要访问 model hub，请在右上角

```

Comment:
Considering use the official product name on the website, it should be: "Model Hub" https://huggingface.co/docs/hub/models-the-hub

Reply:
sure